plugins/inputs/jenkins/README.md

```diff
@@ -57,8 +57,9 @@ This plugin does not require a plugin on jenkins and it makes use of Jenkins API
 
 ### Metrics:
 
-- jenkins_node
+- jenkins
   - tags:
+    - host
```

Comment:
Unless there is a Jenkins specific `host` tag I overlooked, `host` is a global tag managed by configuration in the agent, and the user could provide `omit_hostname=true` having it not appear. So I am not sure `host` belongs in the plugin specific README.md.